src/Notes/Notes.php

```diff
@@ -273,15 +269,10 @@ public static function possibly_delete_marketing_notes( $old_value, $value ) {
 	 */
 	public static function possibly_delete_survey_notes() {
 		$data_store = self::load_data_store();
-		$notes      = $data_store->get_notes(
-			array(
-				'type'   => array( Note::E_WC_ADMIN_NOTE_SURVEY ),
-				'status' => array( 'actioned' ),
-			)
-		);
+		$note_ids   = $data_store->get_notes_with_type( Note::E_WC_ADMIN_NOTE_MARKETING );
```

Comment:
Probably problematic, since this one is a hard delete. The old version wouldn't have deleted rows with an unactioned status, and this one would delete all of them. I'm not sure off the top of my head why we'd want to hard delete only actioned notes, but that's definitely a change.